test/system/700-play.bats

```diff
@@ -531,3 +531,70 @@ spec:
     run_podman pod rm -a -f
     run_podman rm -a -f
 }
+
+@test "podman kube play - multi-pod YAML" {
+    skip_if_remote "service containers only work locally"
+
+    # Create the YAMl file
+    yaml_source="$PODMAN_TMPDIR/test.yaml"
+    cat >$yaml_source <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: pod1
+  name: pod1
+spec:
+  containers:
+  - command:
+    - top
+    image: $IMAGE
+    name: ctr1
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: pod2
+  name: pod2
+spec:
+  containers:
+  - command:
+    - top
+    image: $IMAGE
+    name: ctr2
+EOF
+    # Run `play kube` in the background as it will wait for the service
+    # container to exit.
+    timeout --foreground -v --kill=10 60 \
+        $PODMAN play kube --service-container=true --log-driver journald $yaml_source &>/dev/null &
```

Comment:
This (journald) will not work on RHEL8 rootless. If RHEL8 is still important, you may want to add `skip_if_journald_unavailable` to the top.